governance/pyth_staking_sdk/src/utils/pool.ts

```diff
@@ -23,11 +23,21 @@ export const extractPublisherData = (
         .map((event) => ({
           epoch: event.epoch,
           apy:
-            (event.y * (event.eventData[index]?.otherRewardRatio ?? 0n)) /
-            1_000_000n,
+            (52 *
+              100 *
+              Number(
+                event.y * (event.eventData[index]?.otherRewardRatio ?? 0n),
+              )) /
+            1_000_000 /
```

Comment:
can this double division be written in a cleaner way?

Reply:
same